doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+
+For a single Tree and sample $r$ if we iterate over all the subsets of $S \subset F \setminus \{i\}$, the interventional TreeSHAP method runs with $O(M2^M)$. Note that there are two paths through the tree of particular interest. The first is the instance path for $x$, and the second is the sampled/reference path for $r$. Computing the Shapley value estimate for the sampled $r$ will involve replacing $x$ with values of $r$ and generating a set of perturbed paths. Instead of iterating over the sets, we sum over the paths. Doing so is faster as many of the routes within the tree have overlapping components. We can compute them all at the same time instead of one by one. Doing this means the Interventional TreeSHAP algorithm obtains $O(LD)$ time complexity.
+
+Applied to a random forest with $T$ trees and using $R$ samples to compute the estimates, we obtain $O(TRLD)$ time complexity. The fact that we can sum over each tree in the random forest results from the linearity property of Shapley values.
+
+**Pros**
+- Very fast for a valuable category of models
+- The Shapley values are fairly distributed among the feature values
+- Shapley values can be easily interpreted and visualized
+- Computes the interventional conditional expectation exactly unlike the path-dependent method
+
+**Cons**
+- Less general as a white box method
+- Requires access to the dataset
+- Typically, slower than the path-dependent method
+
+### Global Feature Attribution
+
+Global Feature Attribution methods aim to show the dependency of model output on a subset of the input features. They are a global insight as it describes the behavior of the model over the entire input space. For instance, Accumulated Local Effects plots obtain graphs that directly visualize the relationship between feature and prediction.
+
+Suppose a trained regression model that predicts the number of bikes rented on a given day depending on the temperature, humidity, and wind speed. A global feature attribution plot for the temperature feature might be a line graph plotted against the number of bikes rented. In the bikes rented case, one would anticipate an increase in rentals up until a specific temperature and then a decrease after it gets too hot.
+
+#### Accumulated Local Effects
+
+| Model-types | Task-types     | Data-types  |
+| ----------- | -------------- | ----------- |
+| Black-box   | Classification | Tabular     |
+|             | Regression     |             |
+
+
+Alibi only provides accumulated local effects plots because of the available global feature attribution methods they give the most accurate insight. Alternatives include Partial Dependence Plots. ALE plots work by averaging the local changes in a prediction at every instance in the data distribution. They then accumulate these differences to obtain a plot of prediction over the selected feature dependencies.
```

Comment:
"of the available global feature attribution methods" - can be cut out

Reply:
Links to PDP (our docs, Molnar, maybe scikit-learn).

Reply:
"accumulated local effects (ALE)" - from then on use ALE elsewhere in the document